security/aws/unencrypted_elb_listeners/elb_unencrypted/aws_elb_encryption.pt

```diff
@@ -0,0 +1,330 @@
+name "AWS App/Network Load Balancers w/Internet-facing Unencrypted Listeners"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report any AWS App/Network Load Balancers w/Internet-facing Unencrypted Listeners. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/elb/clb_unused) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
short description: Report on AWS Unencrypted ELB Listeners (ALB/NLB) and terminate after approval. 
Fix path to readme.